test/parallel/test-console-sync-write-error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
The main thing that I'm hoping for is that this would fix the issue reported in https://github.com/nodejs/node/issues/831